src/shared/ui/Dropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC, useMemo } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { DropdownElement } from '.';
+
+type DropdownProps = {
+  placeholder: string;
+  items: Array<number | string>;
+  value?: number | string;
+  onValueChange: (value: number | string) => void;
+};
+
+const Dropdown: FC<DropdownProps> = ({
+  placeholder,
+  value = '',
+  onValueChange,
+  items,
+}) => {
+  const mappedItems = useMemo(
+    () => items.map(item => ({ label: item, value: item })),
```

Comment:
Please define the following type:
```
type LabeledValue<TValue extends string | number = string> = {
  label: string;
  value: TValue;
};
```
`items` should already be of this type. So you don't have to map them here.
`value` should be `TValue`

`TValue` should come from `Dropdown` function

Reply:
Done